### PR TITLE
Hoan - uneject

### PR DESCRIPTION
**Pull request by hoanphungt**

### CODE REVIEW

**Comment by hoanphungt** [comment]
@remcogerlich 
Hi Remco,

At the end, I pulled the master of the existing lizard management client to the new one (adding `--allow-unrelated-histories`) to solve the problem that I couldn't create a PR to merge the new app to the old one. 

I also added some files:

1. setupProxy.js
2. makedist.sh in /scripts directory
3. Change the build script in package.json to: `mkdir -p dist/ && rm -rf dist/*.zip && react-scripts build && scripts/makedist.sh`. However, I still have problem when running `yarn build` here (`permission denied` at scripts/makedist.sh).
4. Change *.css to *.module.css

Hopefully, everything is gonna work correctly (it worked on my laptop at least :) )

Could you have another look and test if everything is also working for you?

File: src/forms/ColorMapInput.tsx [hunk 1]
```diff
@@ -7,18 +7,18 @@ import SelectBox, { choicesT } from "./SelectBox";
...
   calculateNewStyleAndOptions,
   optionsHasLayers,
-  getColorMapFromStyle,
-  getColorMinFromStyle,
-  getColorMaxFromStyle,
-  getStyleFromOptions,
+  // getColorMapFromStyle,
+  // getColorMinFromStyle,
+  // getColorMaxFromStyle,
+  // getStyleFromOptions,
```
Comment (remcogerlich) [inline src/forms/ColorMapInput.tsx:21]: Why is this commented out? Seems unreleated to the unejecting. If they are unused imports then they should be removed, not commented.

File: src/forms/ColorMapInput.tsx [hunk 2]
```diff
@@ -202,10 +202,10 @@ class ColorMapInput extends Component<ColorMapProps & InjectedIntlProps, ColorMa
...
       colorMaps,
       value,
-      valueChanged,
-      validated,
-      placeholder,
-      wizardStyle,
+      // valueChanged,
+      // validated,
+      // placeholder,
+      // wizardStyle,
```
Comment (remcogerlich) [inline src/forms/ColorMapInput.tsx:208]: Same

File: src/forms/DurationField.tsx [hunk 3]
```diff
@@ -106,11 +106,11 @@ export default class DurationField extends Component<DurationFieldProps, {}> {
...
       name,
-      placeholder,
+      // placeholder,
       value,
-      validated,
-      valueChanged,
-      handleEnter,
+      // validated,
+      // valueChanged,
+      // handleEnter,
```
Comment (remcogerlich) [inline src/forms/DurationField.tsx:113]: Same

File: src/forms/TextArea.tsx [hunk 4]
```diff
@@ -23,10 +23,10 @@ export default class TextArea extends Component<TextAreaProps, {}> {
       name,
       placeholder,
       value,
-      validated,
-      valueChanged,
+      // validated,
+      // valueChanged,
       readOnly,
-      handleEnter
+      // handleEnter
```
Comment (remcogerlich) [inline src/forms/TextArea.tsx:29]: Same

File: src/forms/TextInput.tsx [hunk 5]
```diff
@@ -24,8 +24,8 @@ export default class TextInput extends Component<TextInputProps, {}> {
       name,
       placeholder,
       value,
-      validated,
-      valueChanged,
+      // validated,
+      // valueChanged,
```
Comment (remcogerlich) [inline src/forms/TextInput.tsx:28]: Same

File: src/forms/WithStep.tsx [hunk 6]
```diff
@@ -36,15 +34,11 @@ class WithStep extends Component<WithStepProps, {}> {
       errors,
       showCheck,
       showErrors,
-      nextStep,
+      // nextStep,
```
Comment (remcogerlich) [inline src/forms/WithStep.tsx:37]: Same

File: src/forms/WithStep.tsx [hunk 7]
```diff
@@ -36,15 +34,11 @@ class WithStep extends Component<WithStepProps, {}> {
...
       selectStep,
       wizardStyle,
       isLastStep
     } = this.props;
-    /* const mustShowClearButton = modelValue !== ""; */
-
-    const mustShowClearButton = true;
-    const resetModelValue = () => null;
```
Comment (remcogerlich) [inline src/forms/WithStep.tsx:47]: Why deleted? Unused?

**Review by remcogerlich — COMMENTED** [review]
" pulled the master of the existing lizard management client to the new one (adding --allow-unrelated-histories) " sounds very scary, but the PR looks very normal.

Just some questions about the commented out / removed stuff, what is up with that?

File: src/forms/WithStep.tsx (hunk 7 above)
Comment (hoanphungt) [reply src/forms/WithStep.tsx:47]: yes, unused

File: src/forms/ColorMapInput.tsx (hunk 1 above)
Comment (hoanphungt) [reply src/forms/ColorMapInput.tsx:21]: unused. I am gonna remove them

**Pull request merged**